specs/eventing/channel.md

```diff
@@ -152,37 +156,20 @@ Each instantiated Channel (ie, Custom Object) SHOULD have an annotation
 indicating which version of the `Channelable` duck type it conforms to. We
 currently have these versions:
 
-1. [v1beta1](https://github.com/knative/eventing/blob/main/pkg/apis/duck/v1beta1/channelable_types.go)
 1. [v1](https://github.com/knative/eventing/blob/main/pkg/apis/duck/v1/channelable_types.go)
 
-So, for example to indicate that the Channel supports v1beta1 duck type, you
-should annotate it like so (only showing the annotations):
+So, for example to indicate that the Channel supports v1 channelable duck type,
```

Comment:
I'm curious if we could rephrase these changes in a way that would convey:
* We understand there are some implementations out there that adhere to v1beta1 duck and express it
* v1beta1 is exactly the same as v1
* Therefore, <words here>

Because the way I read this sentence coupled with the rest of the changes, there is no versioning